src/hangman/commands/handlers/start-new-game.handler.ts

```diff
@@ -1,29 +1,28 @@
 import { CommandHandler, ICommandHandler } from '@nestjs/cqrs';
 import { StoreEventPublisher } from '@peterdijk/nestjs-eventstoredb';
-
-import { StartNewGameCommand } from '../Commands/StartNewGame.command';
 import { Logger } from '@nestjs/common';
-import { GamesRepository } from '../../Domain/Repositories/GamesRepository';
+
+import { Game } from '../../models/game.model';
+import { GamesRepository } from '../../repository/game.repository';
+import { StartNewGameCommand } from '../impl/start-new-game.command';
 
 @CommandHandler(StartNewGameCommand)
 export class StartNewGameCommandHandler
   implements ICommandHandler<StartNewGameCommand> {
   private readonly logger = new Logger(StartNewGameCommandHandler.name);
 
   constructor(
-    private publisher: StoreEventPublisher,
     private readonly repository: GamesRepository,
+    private publisher: StoreEventPublisher,
   ) {}
 
-  async execute({ data, uuid }: StartNewGameCommand) {
-    const { playerId, wordToGuess, maxGuesses } = data;
+  async execute(command: StartNewGameCommand) {
+    this.logger.log(`StartNewGameCommand...`);
+    const { data, uuid } = command;
 
-    const game = this.publisher.mergeObjectContext(
-      await this.repository.startNewGame(
-        { playerId, wordToGuess, maxGuesses },
-        uuid,
-      ),
-    );
+    const GameModel = this.publisher.mergeClassContext(Game);
```

Comment:
I don't agree with changing the naming of Aggregate to Model. 
Within the Games domain, the game is the aggregate. 
A model is more a ORM entity

Reply:
Fair enough. I agree here. Again, just following the naming convention in the other CQRS example project.